.github/workflows/build_and_run_selected_test.yml

```diff
@@ -0,0 +1,117 @@
+name: Build and Run Selected Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      test-selection:
+        description: 'Specification of selected test(s), as accepted by pytest -k'
+        required: true
+        type: string
+      run-count:
+        description: 'Number of test runs to perform'
+        required: true
+        type: number
+      archs:
+        description: 'Archs to run tests on, e. g.: x64, arm64'
+        default: 'x64, arm64'
```

Comment:
- Let's convert this to a JSON array to make it easy to use in the matrix, i.e. '["x64", "arm64"]'
- Let's use only x64 by default.

Reply:
I chose that format to simplify the specification for human users but will change it to accept valid JSON.